extensions/binding/org.eclipse.smarthome.binding.mqtt.generic/src/main/java/org/eclipse/smarthome/binding/mqtt/generic/internal/homie300/NodeInstance.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.binding.mqtt.generic.internal.homie300;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * TODO Homie specification needs a revised documentation and handling of instances.
```

Comment:
Is this class needed then?

Reply:
Technically this is not homie standard compliant without this support, but I'm not feeling like implementing it right now. Some kind of documented "we don't do this on purpose" should be left here I guess

Reply:
okay for me.